host/js/src/common/sf_host.ts

```diff
@@ -12,11 +13,11 @@ function strace<A extends unknown[], R>(name: string, fn: Fn<A, R>, asyncify: As
     const result: any = fn(...args);
 
     // TODO: control this from variable
-    // if (asyncify.getState() === AsyncifyState.Normal) {
-    //   console.debug(`host: [strace] ${name}(${args}) = ${result}`);
-    // } else {
-    //   console.debug(`host: [strace] ${name}(${args}) = <async suspended (${result})>`);
-    // }
+    if (asyncify.getState() === AsyncifyState.Normal) {
+      console.debug(`host: [strace] ${name}(${args}) = ${result}`);
+    } else {
+      console.debug(`host: [strace] ${name}(${args}) = <async suspended (${result})>`);
+    }
```

Comment:
Is it left uncommented intentionally? It seems not to obey DEV_LOG

Reply:
Yes, I uncommented this but removed the strace call at the end of the file, so this is now never called. I would leave it in though in case we need that strace debugging during development.